scripts/screens/ProfileScreen.py

```diff
@@ -871,11 +871,30 @@ def generate_column2(self, the_cat):
             # NEWLINE ----------
             output += "\n"
 
-        if the_cat.status.is_other_clancat:
+        if the_cat == game.clan.instructor:
+            output += i18n.t(f"general.guide")
+            output += "\n"
+
+        # add group name
+        if the_cat.dead and (
+            the_cat.status.is_clancat or the_cat.status.is_former_clancat
+        ):
+            if the_cat == game.clan.instructor:
+                # these guys are special, we won't name a group for them
+                output += f"{i18n.t(f'general.past_no_group')} "
```

Comment:
This is forcing localization to follow English word order. Could you do what you did for `general.past_group`? also, just phrasing but I personally think "past" could be better served by the word "former", like we used to use for "former Clancat" ;)